prow/scripts/build-kyma-artifacts.sh

```diff
@@ -40,4 +40,12 @@ gsutil cp "${ARTIFACTS}/kyma-installer-local.yaml" "${KYMA_ARTIFACTS_BUCKET}/${D
 
 gsutil cp "/home/prow/go/src/github.com/kyma-project/kyma/installation/resources/tiller.yaml" "${KYMA_ARTIFACTS_BUCKET}/${DOCKER_TAG}/tiller.yaml"
 
+gsutil cp "${ARTIFACTS}/kyma-installer-cr.yaml" "${KYMA_ARTIFACTS_BUCKET}/${DOCKER_TAG}/kyma-installer-cr.yaml"
+gsutil cp "${ARTIFACTS}/installer.yaml" "${KYMA_ARTIFACTS_BUCKET}/${DOCKER_TAG}/installer.yaml"
+gsutil cp "${ARTIFACTS}/installer-cr-cluster.yaml" "${KYMA_ARTIFACTS_BUCKET}/${DOCKER_TAG}/installer-cr-cluster.yaml"
+gsutil cp "${ARTIFACTS}/installer-cr-cluster-compass.yaml" "${KYMA_ARTIFACTS_BUCKET}/${DOCKER_TAG}/installer-cr-cluster-compass.yaml"
+gsutil cp "${ARTIFACTS}/installer-cr-cluster-compass-dependencies.yaml " "${KYMA_ARTIFACTS_BUCKET}/${DOCKER_TAG}/installer-cr-cluster-compass-dependencies.yaml"
```

Comment:
I don't think these two files are generated in `release-generate-kyma-installer-artifacts.sh` script in Kyma repo. Compass related CR's were recently removed.

Reply:
fixed.